README.md

```diff
@@ -1,10 +1,11 @@
 # Ruby Project
 
+[Be a Ruby](https://www.bearuby.org/) is a local non-profit organization that distributes items that they collect through donations to people in need in the Grand Rapids community. They currently have limited inventory tracking, which makes it difficult to see what items need to be collected. Our mission is to create Ruby, a custom mobile application that will provide an intuitive interface for them to log and update their stock. To increase security and reduce errors when updating stock, the app will have administrator and site roles that limit access to reading and writing to the stock log.
```

Comment:
Update "distributes items that they collect through donations" -> "distributes food and clothing donations"

We need to clarify in the first paragraph that this is only for tracking items in the trailers like utensils, to go boxes, and cups, not all of their clothing donations. In the first paragraph we say they have limited stock tracking, in the second we say that they have no stock tracking. Probably replace the first paragraph second sentence with the first sentence of the second paragraph.

Reply:
First paragraph last sentence: log -> database

Reply:
Update 3rd sentence to: "Our mission is to create Ruby, a custom mobile application, that will provide an intuitive interface for them to update their trailer inventories"